.github/workflows/main.yaml

```diff
@@ -216,12 +216,13 @@ jobs:
           echo "codegen-units = 1" >> Cargo.toml
           echo "rpath = false" >> Cargo.toml
           make build-wasmer-headless-minimal
-        if: needs.setup.outputs.DOING_RELEASE == '1'
       - name: Copy target binaries
         run: |
           mkdir -p target/release
           cp target/${{matrix.target}}/release/wasmer* target/release
-          cp target/${{matrix.target}}/release/libwasmer* target/release
+          if find target/${{matrix.target}}/release -type f -name 'libwasmer*' -maxdepth 1 | grep -q "."; then
```

Comment:
Why would there ever be a libwasmer without a `.` in the name? Maybe this could have a comment?

Reply:
This step will be removed:  the `TARGET` variable in the Makefile will do the job :)